debounce/simple_debouncer_test.go

```diff
@@ -52,11 +52,12 @@ func TestSimpleDebouncer(t *testing.T) {
 		}
 	})
 }
+
 func waitTillChannelIsEmpty(ch chan struct{}) {
 	for {
 		if len(ch) == 0 {
 			return
 		}
-		runtime.Gosched()
+		time.Sleep(100 * time.Millisecond)
```

Comment:
Please remove it from this PR, if there is a reason for it here, please create a separate PR for it.

Reply:
Ok. 
This is reason of the failing pipelines.

Reply:
PR #344 was created.
Changes removed from this PR.